k-distribution/pl-tutorial/1_k/4_imp++/lesson_3/README.md

```diff
@@ -31,99 +31,6 @@ the `krun` tool with the option `--search`. For example:
 
```

Comment:
We've removed almost everything from this lesson (the title is "Tagging; Transition Kompilation Option", and we no longer explain tagging or `--transition`). 

If this is still the case after addressing the other comments I've left, we should consider deleting the lesson outright and absorbing the remaining content (defining the variable increment rule + describing `--search`) into other lessons.